patroni/dcs/__init__.py

```diff
@@ -1557,42 +1571,45 @@ def __get_patroni_cluster(self, path: Optional[str] = None) -> Cluster:
-        cluster = self._load_cluster(path, self._cluster_loader)
+        cluster = self._load_cluster(path, self._patroni_cluster_loader)
         if TYPE_CHECKING:  # pragma: no cover
             assert isinstance(cluster, Cluster)
         return cluster
 
-    def is_citus_coordinator(self) -> bool:
-        """:class:`Cluster` instance has a Citus Coordinator group ID.
+    def is_coordinator_cluster(self) -> bool:
+        """:class:`Cluster` instance has a Coordinator group ID.
 
-        :returns: ``True`` if the given node is running as Citus Coordinator (``group=0``).
+        :returns: ``True`` if the given node is running as Coordinator.
         """
-        return self._citus_group == str(CITUS_COORDINATOR_GROUP_ID)
+        return self._group is not None and self._group == str(self.coordinator_group_id)
 
-    def get_citus_coordinator(self) -> Optional[Cluster]:
-        """Load the Patroni cluster for the Citus Coordinator.
+    def get_coordinator_cluster(self) -> Optional[Cluster]:
+        """Load the Coordinator cluster of the formation cluster.
 
           .. note::
-              This method is only executed on the worker nodes (``group!=0``) to find the coordinator.
+              This method is only executed on the worker nodes to find the coordinator.
 
-        :returns: Select :class:`Cluster` instance associated with the Citus Coordinator group ID.
+        :returns: Select :class:`Cluster` instance associated with the Coordinator group ID.
         """
         try:
-            return self.__get_patroni_cluster(f'{self._base_path}/{CITUS_COORDINATOR_GROUP_ID}/')
+            return self.__get_patroni_cluster(f'{self._base_path}/{self.coordinator_group_id}/')
         except Exception as e:
-            logger.error('Failed to load Citus coordinator cluster from %s: %r', self.__class__.__name__, e)
+            logger.error('Failed to load coordinator cluster from %s: %r', self.__class__.__name__, e)
         return None
 
-    def _get_citus_cluster(self) -> Cluster:
-        """Load Citus cluster from DCS.
+    def _get_formation_cluster(self) -> Cluster:
+        """Load formation cluster from DCS.
 
-        :returns: A Citus :class:`Cluster` instance for the coordinator with workers clusters in the `Cluster.workers`
+        :returns: A multi :class:`Cluster` instance for the coordinator with workers clusters in the `Cluster.workers`
                   dict.
         """
-        groups = self._load_cluster(self._base_path + '/', self._citus_cluster_loader)
+        groups = self._load_cluster(self._base_path + '/', self._formation_cluster_loader)
         if TYPE_CHECKING:  # pragma: no cover
             assert isinstance(groups, dict)
-        cluster = groups.pop(CITUS_COORDINATOR_GROUP_ID, Cluster.empty())
+        if self.coordinator_group_id is None:
+            cluster = Cluster.empty()
+        else:
+            cluster = groups.pop(self.coordinator_group_id, Cluster.empty())
```

Comment:
All these checks are unnecessary.
```python
In [1]: a = {1: 2, 3: 4}

In [2]: a.pop(None, 'empty')
Out[2]: 'empty'

In [3]: a
Out[3]: {1: 2, 3: 4}
```
```suggestion
        cluster = groups.pop(self.coordinator_group_id, Cluster.empty())
```